content/applications/inventory_and_mrp/manufacturing/workflows/work_center_time_off.rst

```diff
@@ -1,57 +1,49 @@
-============================================
-Make work centers unavailable using Time Off
-============================================
+====================
+Work center time off
+====================
+
+.. |MO| replace:: :abbr:`MO (manufacturing order)`
 
 In Odoo, *work centers* are used to carry out manufacturing operations at specific locations.
 However, if a work center cannot be used for some reason, work orders begin to pile up at the work
 center until it is operational again.
 
-As a result, it is necessary to make the work center unavailable in Odoo so that the platform routes
-new work orders to alternative work centers that are operational. Using Odoo *Time Off*, it is
+As a result, it is necessary to make the work center unavailable in Odoo so new work orders are
+routed to alternative work centers that are operational. Using Odoo's **Time Off** app, it is
 possible to designate a work center as being unavailable for a set period of time. Doing so ensures
-that manufacturing operations can continue until the impacted work center is available again.
+manufacturing operations can continue until the impacted work center is available again.
 
 Configuration
 =============
 
 Before a work center can be designated as unavailable, the Odoo platform must be properly
-configured. First, it is necessary to enable :ref:`developer mode <developer-mode>`. This allows the
-:guilabel:`Time Off` smart button to appear on each work center's :guilabel:`Working Hours` pop-up
-window.
-
-Enable developer mode by going to :menuselection:`Settings`, scrolling to the bottom of the page,
-and clicking :guilabel:`Activate the developer mode` under the :guilabel:`Developer Tools` heading.
+configured. First, it is necessary to enable :doc:`developer mode
+<../../../general/developer_mode>`. This allows the *Time Off* smart button to appear on each work
+center's *Working Hours* page.
```

Comment:
Nitpick, but I prefer using the developer-mode reference instead of doc :)
```suggestion
configured. First, it is necessary to enable :ref:`developer mode <developer-mode>`. This allows the
*Time Off* smart button to appear on each work center's *Working Hours* page.
```